public/css/stylesheet.css

```diff
@@ -229,3 +229,73 @@ Aligns Main Menu under button
   justify-content: space-between;
 }
 /*profile page css end*/
+.form-box {
+    margin-top: 40px;
+    margin-right: 50px;
+    padding-right: 65px;
+   
+}
+.form-top {
+    overflow: hidden;
+    padding: 0 15px 15px 25px;
+    background: #444;
+    background: rgba(0, 0, 0, 0.35);
```

Comment:
Change the opacity to 0.2, it makes it a bit lighter and more consitant with the rest of the website.